website/blog/2023-07-12-release-1.2.md

```diff
@@ -0,0 +1,166 @@
+- Only show empty screen when no pods ([PR #2929](https://github.com/containers/podman-desktop/pull/2929))
+- Do not redirect to /pods when deleting pod in containerlist ([PR #2963](https://github.com/containers/podman-desktop/pull/2963))
+- Bulk delete on pods should call the pod deletion ([PR #2979](https://github.com/containers/podman-desktop/pull/2979))
+- Update nodejs version to 18 in .nvmrc to fix yarn install failure ([PR #2989](https://github.com/containers/podman-desktop/pull/2989))
+- Website check targets ([PR #2996](https://github.com/containers/podman-desktop/pull/2996))
+- Don't show exception to user ([PR #3034](https://github.com/containers/podman-desktop/pull/3034))
+- Interpret arguments given to info command for example ([PR #3015](https://github.com/containers/podman-desktop/pull/3015))
+- Change defaults for Podman machine ([PR #3061](https://github.com/containers/podman-desktop/pull/3061))
+- Normalize development/production folders path ([PR #3113](https://github.com/containers/podman-desktop/pull/3113))
+- Calculate machine socket path for linux ([PR #3070](https://github.com/containers/podman-desktop/pull/3070))
+- Replace backslash with slash so to support rendering on Windows (#3120) ([PR #3122](https://github.com/containers/podman-desktop/pull/3122))
+- Keep stdout and stderr in the error object for Docker Desktop extensions ([PR #3014](https://github.com/containers/podman-desktop/pull/3014))
+- Mark task completed if there's a failure ([PR #3016](https://github.com/containers/podman-desktop/pull/3016))
+- Handle invalid kubeconfig file ([PR #3129](https://github.com/containers/podman-desktop/pull/3129))
+- Do not remove registries when podman extension is stopping ([PR #3136](https://github.com/containers/podman-desktop/pull/3136))
+- Warning should be amber ([PR #3153](https://github.com/containers/podman-desktop/pull/3153))
+- Load user extensions from plugins as removable ([PR #3152](https://github.com/containers/podman-desktop/pull/3152))
+- Images with spaces in entrypoints or commands fail to start ([PR #3161](https://github.com/containers/podman-desktop/pull/3161))
+- Scrolling offscreen when clicking checkbox ([PR #3178](https://github.com/containers/podman-desktop/pull/3178))
+- Avoid messagebox expanding offscreen ([PR #2778](https://github.com/containers/podman-desktop/pull/2778))
+- Release-notes-generator run failure ([PR #2752](https://github.com/containers/podman-desktop/pull/2752))
+- Unable to do a new build if the previous one failed ([PR #2721](https://github.com/containers/podman-desktop/pull/2721))
+
+---
+
+## Community Thank You
+
+🎉 We’d like to say a big thank you to everyone who helped to make Podman Desktop even better. In this
+sprint we received pull requests from new contributors:
+
+- [tuckerrc](https://github.com/containers/podman-desktop/pull/2588) for adding the ability to rename an image in [PR #2588](https://github.com/containers/podman-desktop/pull/2588).
+
```

Comment:
I know he's not 'new', but afbjorklund did PR #2996, #3070, #3051, #3034 and feels like we should still mention him. Maybe we can go back to a more general community thank you?

evanshortiss did #2884. He is Red Hat, but IMHO also qualifies as 'community contribution'.

Reply:
Ah thank you! Was going by the automated release notes that only showed tucker. I'll update it. Thank you!